lewis/scripts/control.py

```diff
@@ -48,7 +48,7 @@ def show_api(remote, object_name):
             raw_value = str(getattr(obj, prop))
             value_lines = raw_value.split('\n')
 
-            current_value = value_lines[0][40:] + (
+            current_value = value_lines[0][:40] + (
```

Comment:
You might be right, I just tried again on a different machine being careful to use the right script. So forget everything I wrote except for "good catch" and add another "good catch".